extensions/skyportal/data/instruments_fritz.yaml

```diff
@@ -537,3 +537,10 @@
   telescope_id: =Keck2
   type: spectrograph
   band: NIR
+- =id: PGIR
+  band: NIR
+  filters:
+  - 2massj
```

Comment:
We are using `sncosmo`, which as far as I can tell does not have `2MASS` filters among built-in bandpasses. You could either submit a PR to `sncosmo` with the filter definitions, or choose a filter that close enough (like, perhaps, `cspjs` or `cspjd`). Please update.

Reply:
See https://sncosmo.readthedocs.io/en/stable/bandpass-list.html#csp